src/dmd/cli.d

```diff
@@ -645,6 +645,9 @@ dmd -cov -unittest myprog.d
                 to the file $(TT profilegc.log) upon program termination.)
             )`,
         ),
+	Option("recursion-limit=<num>",
```

Comment:
It would be better to have `template` in the option name.

Reply:
Ahh, but it also controls recursive type depth, so template may not be such an accurate description afterall.

Reply:
Removed the inaccuracy